src/Contract/Transaction.purs

```diff
@@ -253,31 +314,69 @@ derive newtype instance Eq BalancedSignedTransaction
 instance Show BalancedSignedTransaction where
   show = genericShow
 
+-- |Like 'balanceAndSignTx, but for more than one transaction.
```

Comment:
In PureScript, `|` is required on each line of doc-comment.

Reply:
Didn't know that.